.github/workflows/test.yml

```diff
@@ -8,6 +8,9 @@ on:
 jobs:
   tests:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node_version: [20, 22, 23]
```

Comment:
@bickelj I suppose it comes down to this: if the dev version has churn before the fork, do we want to be continuously making sure we support all versions of the churn.

Reply:
> I want to take it even one step further and add the exact expected production node version.

This would mean we don't benefit from the latest security patches without a manual developer action.  IMHO let's stick with the node ecosystem's general standard of major version being the level that we specify for test / deploy.

Reply:
Your premise is incorrect, though. We lack the ability to get the latest security patches even if we wanted them when using DO's buildpack. We're stuck on an old minor version. We can do better than the general node population here : )